readers/hdfs/src/main/resources/logback.xml

```diff
@@ -9,6 +9,11 @@
     <logger name="org.apache.kafka.common.utils.AppInfoParser" level="INFO" />
     <logger name="org.apache.kafka.clients.consumer.ConsumerConfig" level="INFO" />
 
+    <logger name="org.apache.hadoop.hdfs.RemoteBlockReader2" level="TRACE" />
```

Comment:
To change log level, change configuration on mesos-configuration

Reply:
This one is only used for local debugging and won't be taken into account

Reply:
I put this in mesos-configuration.
I don't understand your second statement.